### PR TITLE
Mafia: Make CS indication more obvious for hosts

### PR DESCRIPTION
**Pull request by ZestOfLife**
Qol for hosts

### CODE REVIEW

**Comment by HoeenCoder** [comment]
Have seen, am communicating in mafia staff discord.

**Comment by Zarel** [comment]
@HoeenCoder bump?

**Comment by HoeenCoder** [comment]
Suggested this in #code-nerds but it got buried and I forgot about this sorry: What about something other than a color indicator since that seems unituitive. Maybe just add a text indicator to the end (eg: `Rolelist (CS):`

File: server/chat-plugins/mafia.ts [hunk 1]
```diff
@@ -1773,9 +1773,9 @@ export const pages: PageTable = {
 					buf += `<p>${game.theme.desc}</p>`;
 				}
 				if (game.noReveal) {
-					buf += `<p><span style="font-weight:bold;">Original Rolelist</span>: ${game.originalRoleString}</p>`;
+					buf += `<p><span style="font-weight:bold;">Original Rolelist ${game.closedSetup ? '(CS)' : ''}</span>: ${game.originalRoleString}</p>`;
```
Comment (whalemer) [inline server/chat-plugins/mafia.ts:1776]: Won't this render as `Original rolelist : whatever, whatever` if it's not a cs?

**Comment by whalemer** [comment]
thanks!

**Comment by Zarel** [comment]
After two months... this certainly deserves congratulations!

🎉

**Pull request merged**